ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ipv6_input_ip_name_servers.yml

```diff
@@ -0,0 +1,13 @@
+---
+type: l2leaf
+
+l2leaf:
+  defaults:
+
+ip_name_servers:
```

Comment:
This data model is for `eos_cli_config_gen`. Test coverage should be added here: https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dns-ntp.yml#L11

Reply:
For `eos_designs` the data model is  `name_servers` and a test should be added here:
https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AVD_LAB.yml#L34.
But for this data model the description must be updated in the schema fragment: https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/name_servers.schema.yml#L16

Reply:
Hey Claus thanks for the feedback. I have removed the test I originally created and added to the tests you suggested.